packages/server-dev/lib/Page.js

```diff
@@ -31,19 +31,24 @@ const Page = ({ auth, Components, config, pageId, resetContext, router, types })
     return <RestartingPage />;
   }
   return (
-    <Client
-      auth={auth}
-      Components={Components}
-      config={{
-        ...config,
-        pageConfig,
-      }}
-      resetContext={resetContext}
-      router={router}
-      stage="dev"
-      types={types}
-      window={window}
-    />
+    <>
+      <Components.Head>
```

Comment:
Is this the right place for this? Shouln't it maybe be in `_document.js`?
Not sure what this is doing, so just asking ignorantly